src/scratchtocatrobat/converter/converter.py

```diff
@@ -1598,8 +1614,22 @@ def _convert_create_clone_of_block(self):
         if len(base_sprite) == 0:
             return catbricks.NoteBrick("Can't convert Clone-Brick with no argument.")
 
+        if base_sprite == "_myself_":
+            print("Clone of ", self.sprite.name)
+            return self.CatrobatClass(self.sprite)
+
         if isinstance(base_sprite, basestring):
-            create_clone_of_brick = self.CatrobatClass(catbase.Sprite(base_sprite))
+            for sprite in self.scene.spriteList:
+                if sprite.name == base_sprite:
```

Comment:
name is a private field of Sprite class. Getter method getName() can be used instead.